governance/polls/Offboard COMP-A - November 15, 2021.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Offboard the COMP-A Vault Type - November 15, 2021
+summary: Signal your support or opposition to offboard the COMP-A vault type.
+discussion_link: https://forum.makerdao.com/t/signal-request-offboarding-matic-comp-aave-and-bal/11184
+vote_type: Plurality Voting
+categories:
+   - Collateral
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2021-11-15T16:00:00
+end_date: 2021-11-18T16:00:00
+---
+# Poll: Offboard the COMP-A Vault Type - November 15, 2021
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the MakerDAO Community. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, November 15 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes: 
+* Lower the COMP-A [Liquidation Penalty (chop)](https://makerdao.world/en/learn/governance/param-liquidation-penalty/) to 0%.
+* Lower the COMP-A [Debt Ceiling (line)](https://makerdao.world/en/learn/governance/param-debt-ceiling) to 0 DAI.
+* Increase the COMP-A [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio/) over 60 days using dss-lerp such that all vaults are liquidated by the end of that period.
+
```

Comment:
Is it worth linking to the thread from Risk with the recommended offboarding parameters? Again can take it or leave it

https://forum.makerdao.com/t/proposed-offboarding-collateral-parameters-2/11548